client/src/templates/Introduction/components/CertChallenge.js

```diff
@@ -89,6 +89,12 @@ const CertChallenge = ({
       )?.show ?? false
     );
 
+    if (!name) {
+      steps.isNameSet = false;
+    } else {
+      steps.isNameSet = true;
+    }
```

Comment:
```suggestion
    steps.isNameSet = !!name;
```

Reply:
Is this new syntax?

Reply:
I don't believe it's that new.

Reply:
@Sembauke We could always make it more fun:
```js
steps.isNameSet = !!!!!!!!!!!!name;
```

_I wonder if there is a performance penalty for doing that 🤔